lib/driverProviders/local.ts

```diff
@@ -113,6 +111,10 @@ export class Local extends DriverProvider {
     if (serverConf.port === undefined) {
       serverConf.port = this.config_.seleniumPort;
     }
+    if (!Array.isArray(serverConf.jvmArgs)) {
+      logger.warn('jvmArgs should be an array!');
```

Comment:
We should throw a ConfigError here OR change the config.ts to say that jvmArgs could be type `string|string[]`.